File: BackEnd/backend/src/main/java/com/fitapp/backend/auth/aplication/dto/response/AuthResponse.java
package com.fitapp.backend.auth.aplication.dto.response;

import java.time.Instant;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthResponse {
    @JsonProperty("token")
    private String token;
    @JsonProperty("refreshToken")
    private String refreshToken;
    @JsonProperty("expiresAt")
    private Instant expiresAt;
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/response/AuthResponse.kt
package com.fitapp.appfit.response

import com.google.gson.annotations.SerializedName

data class AuthResponse(
    @SerializedName("token") val token: String,
    @SerializedName("refreshToken") val refreshToken: String,
    @SerializedName("expiresAt") val expiresAt: Long
)